### PR TITLE
Story(CCLS-2099) Post assessment endpoint

### PR DESCRIPTION
**Pull request by PhilDigitalJustice**
includes amendments to the PUT request to update existing assessments

### CODE REVIEW

File: assessment-service/src/main/java/uk/gov/laa/ccms/caab/assessment/entity/AuditTrail.java [hunk 1]
```diff
@@ -2,8 +2,12 @@
 
 import jakarta.persistence.Column;
 import jakarta.persistence.Embeddable;
+import java.io.Serializable;
 import java.util.Date;
 import lombok.Data;
```
Comment (farrell-m) [inline assessment-service/src/main/java/uk/gov/laa/ccms/caab/assessment/entity/AuditTrail.java:7]: No longer used?

Comment (farrell-m) [reply assessment-service/src/main/java/uk/gov/laa/ccms/caab/assessment/entity/AuditTrail.java:7]: Also the same for the other entities

**Review by farrell-m — CHANGES_REQUESTED** [review]
Just a small comment, otherwise all looks good.

File: assessment-service/src/main/java/uk/gov/laa/ccms/caab/assessment/entity/AuditTrail.java (hunk 1 above)
Comment (PhilDigitalJustice) [reply assessment-service/src/main/java/uk/gov/laa/ccms/caab/assessment/entity/AuditTrail.java]: Amended

**Review by farrell-m — APPROVED** [review]
👍

**Pull request merged**